auth0_flutter/lib/src/web/js_interop.dart

```diff
@@ -165,24 +166,25 @@ class PopupLoginOptions {
 
 @JS()
 @anonymous
-class PopupConfigOptions {
-  external dynamic get popup;
-  external int? get timeoutInSeconds;
+extension type PopupConfigOptions._ (JSObject _ ) implements JSObject  {
+  external JSAny  popup;
```

Comment:
This should be `JSAny?` and there is a space too many. The `get` is also gone here?